.builds/archlinux.yml

```
@@ -0,0 +1,16 @@
+image: archlinux
+packages:
+  - meson
+  - wayland
+  - wayland-protocols
+  - scdoc
+sources:
+  - https://github.com/swaywm/swayidle
+tasks:
+  - setup: |
+      cd swayidle
+      meson build
```

I'd rather be explicit about this, so we pick up on anything which accidentally disables the feature.

```suggestion
      meson -Dlogind=enabled build
```

---

Maybe even `-Dauto-features=enabled` just in case we add some more in the future?